pytorch_to_returnn/torch/tensor.py

```diff
@@ -145,3 +145,8 @@ def __mul__(self, other):
   def __truediv__(self, other):
     from .nn.functional import truediv
     return truediv(self, other)
+
+
+class LongTensor(Tensor):
+  def __init__(self, **kwargs):
+    super(LongTensor, self).__init__(dtype="float64", **kwargs)
```

Comment:
Wrong dtype. Should be int64, right?
```suggestion
    super(LongTensor, self).__init__(dtype="int64", **kwargs)
```

Reply:
Oh yes, that's right of course.